examples/window-app/common/window-app.zap

```diff
@@ -2464,7 +2464,7 @@
               "reportableChange": 0
             },
             {
-              "name": "BatteryPercentRemaining",
+              "name": "BatPercentRemaining",
```

Comment:
Following your logic 2 diffs upper why didn't you do bat percent remaining 
I don't follow here

Reply:
Sorry, not sure what's your concern here.